plotnine/tests/test_geom_boxplot.py

```diff
@@ -21,7 +26,8 @@ class TestAesthetics:
          geom_boxplot(aes(y='y'), size=2) +
          geom_boxplot(df[:2*m], aes(y='y+25', fill='x'), size=2) +
          geom_boxplot(df[2*m:], aes(y='y+30', color='x'), size=2) +
-         geom_boxplot(df[2*m:], aes(y='y+55', linetype='x'), size=2)
+         geom_boxplot(df[2*m:], aes(y='y+55', linetype='x'), size=2) +
+         geom_boxplot(df[:2*m], aes(y='y+55', weight='weight'), size=2)
```

Comment:
Can you create a separate test case specifically for the weight one that is intuitive to the reader (at least with some concentration). For example using a dataframe like this:

```python
# Data with two groups,
# - 'a' has no weights
# - 'b' has weights that should make the resulting
#    boxplot same as that of 'a'
df = pd.DataFrame({
    'x': list('a'*10 + 'b'*5),
    'y': np.hstack([
        [1, 2, 2, 3, 3, 3, 4, 4, 4, 4, 20],
        [1, 2, 3, 4, 20]  # 20 should be an outlier
    ]),
    'w': np.hstack([
        np.ones(10),
        [1, 2, 3, 4, 1]
    ])
})
```
I actually don't know if it works, but I hope you get what I am trying to do!